example_notebooks/7_ALS_toyblock_demo.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>"Besides the typical properties, such as <code>beamDivergence_rad</code> or <code>accuracy</code>, we see something special for this scanner: The scanner has three channels with a different <code>beamOrigin</code> each. This way, the multi-channel laser scanner Livox Mid-100 is simulated, which has three LiDAR sensors with overlapping fields of view." </p><p> </p><p>We could even add an image from the <a href="https://www.livoxtech.com/3296f540ecf5458a8829e01cf429798e/downloads/20190530/Livox%20Mid-100%20Quick%20Start%20Guide%20multi%20v1.4.pdf" target="_blank">datasheet</a> ,page 3, or from the <a href="https://www.livoxtech.com/de/mid-40-and-mid-100" target="_blank">livox website</a>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>